v-next/hardhat/src/internal/cli/telemetry/sentry/anonymizer.ts

```diff
@@ -212,15 +270,20 @@ export class Anonymizer {
     const nomicFoundationPath = path.join("node_modules", "@nomicfoundation");
     const ignoredOrgPath = path.join("node_modules", "@ignored");
     const hardhatPath = path.join("node_modules", "hardhat");
+
     filename = filename.toLowerCase();
 
     return (
-      filename.startsWith(nomicFoundationPath) ||
-      filename.startsWith(ignoredOrgPath) ||
-      filename.startsWith(hardhatPath)
+      filename.includes(nomicFoundationPath) ||
+      filename.includes(ignoredOrgPath) ||
+      filename.includes(hardhatPath)
```

Comment:
I changed this to includes as we are operating against the absolute path, rather than the anonymized version of the path that removed everything before the project directory.